tools/acrn-manager/README.rst

```diff
@@ -93,6 +94,25 @@ Use the ``stop`` command to stop one or more running VM:
 
    # acrnctl stop vm-yocto vm1-14:59:30 vm-android
 
+RESCAN BLOCK DEVICE
+===================
+
+Use the ``blkrescan`` command to trigger a rescan of
+virtio-blk device by guest VM, inorder to revalidate and 
+update the backend file.
+
+.. code-block:: none
+
+   # acrnctl blkrescan vmname slot,newfilepath
```

Comment:
Could we add a real example, including the output that is generated on the terminal when running this command?

Reply:
done. Regarding the acrnctl blkrescan command, it doesn't generate any output when executed but have added a real example